agenta-backend/agenta_backend/services/evaluators_service.py

```diff
@@ -232,17 +315,18 @@ def auto_custom_code_run(
     if not isinstance(output, str):
         output = output.get("data", "")
     try:
-        result = sandbox.execute_code_safely(
-            app_params=app_params,
-            inputs=inputs,
-            output=output,
-            correct_answer=data_point.get(
-                "correct_answer", None
-            ),  # for backward compatibility
-            code=settings_values["code"],
-            datapoint=data_point,
+        correct_answer = get_correct_answer(data_point, settings_values)
+        inputs = {
+            "app_config": app_params,
+            "prediction": output,
+            "ground_truth": correct_answer,
+        }
+        response = await custom_code_run(
```

Comment:
This could affect existing custom code evaluators, right ?

Reply:
Nope.